engine-mr/src/main/java/org/apache/kylin/engine/mr/common/JobInfoConverter.java

```diff
@@ -61,6 +61,20 @@ public static JobInstance parseToJobInstanceQuietly(CheckpointExecutable job, Ma
         }
     }
 
+    public static JobInstance parseToJobInstanceQuietly(AbstractExecutable job, Map<String, Output> outputs) {
+        try {
+            if (job instanceof CheckpointExecutable) {
+                return parseToJobInstance((CheckpointExecutable)job, outputs);
+            } else if (job instanceof CubingJob) {
+                return parseToJobInstance((CubingJob)job, outputs);
+            } else {
+                return null;
+            }
+        } catch (Exception e) {
+            logger.error("Failed to parse job instance: uuid={}", job, e);
+            return null;
+        }
+    }
```

Comment:
Hi, @guangxuCheng . Actually this method has replaced `parseToJobInstanceQuietly(CubingJob job, Map<String, Output> outputs)` and `parseToJobInstanceQuietly(CheckpointExecutable job, Map<String, Output> outputs)`, so should  the replaced method and their UT be removed? And maybe UT for this new method should be added.

Reply:
@zhangayqian Thanks for reviewing. A nice finding,I will remove the two method. 
The new method is to combine `parseToJobInstanceQuietly(CubingJob job, Map<String, Output> outputs)` and `parseToJobInstanceQuietly(CheckpointExecutable job, Map<String, Output> outputs)`, so the ut about the two method is suitable for new method.

Reply:
I force-pushed the branch to rebase master and remove the two methods.

Reply:
Thanks for your update in time!